### ISSUE TITLE
DP Assignments Use Case

### ISSUE THREAD

**Issue by koushik-ac** [beezlabs-org/automation-dev#62]
All the development and Internal Testing has been completed. We are waiting for UAT to begin post MEC and Q00 migration gets completed.

**Comment by karthikeyan23** [comment]
Q00 is down so, the testing is planned for Wednesday

**Comment by dominicceo** [comment]
@koushik-ac can you help scheduling a call already and close it please as we have to book their calendar in advance to see if they have any OOO

**Comment by koushik-ac** [comment]
Since we are not allowed to schedule calls directly with End Users, Bharath has asked Sergio to setup a call. A UAT call has been scheduled now for Wednesday late night.

**Comment by koushik-ac** [comment]
We have completed the UAT call yesterday, and the run was successful. They have asked for a change in the File Name to include the date in the name of the file. @AnanthaKrishnan3646 - the final output file should contain the following name: DP_Assignment_DD_MM_YYYY.XLSX. Let me know if you have any doubts.

The team has given a new Share Point location to upload the output data. Currently using the reusable bot, I am not able to upload the file and getting an error. Need a C# developer to help me with this.

**Comment by karthikeyan23** [comment]
@AnanthaKrishnan3646  to complete the file name change today and complete the testing by tomorrow

**Comment by karthikeyan23** [comment]
#issue in uploading the files to the newly provided Sharepoint folder, we need to check and confirm. Worst case customer has agreed to use the old path

**Comment by koushik-ac** [comment]
All the issues have been resolved and we have prepared the UAT document for this use case and sent it for Approval.
@Govi02 and @AMANSARTHAK19 - We need to prepare TS for this use case.

**Comment by Dean13b** [comment]
Received UAT approval. Awaiting FS approval for TR movement. Planned movement is by 20th Sept

**Comment by koushik-ac** [comment]
We have to setup a call with the new Functional contact as Luciano role has been changed.

**Comment by karthikeyan23** [comment]
@dominicceo  I have been informed that Luci is moving out and we have to work with a new person on this. It will be correct and effective if Luci does the transition., we doing the transition is a waste of effort from outside and it will less effective. What is Juan thought on this?

**Comment by dominicceo** [comment]
We are trying to address this, as per Juan going back to Luci has been denied as we have been told not to reach out to him anymore, we have written to the new RTR person and we might not need the consutant badnwidth to have been spent extra on this, we are trying to close this use case with very less intervention, @koushik-ac my only question is have you ever spent time with Luci before on this use case?

**Comment by karthikeyan23** [comment]
@dominicceo  but whether its Consultant effort or not its a waste of effort for Beezlabs, its a very normal common sense thing that the person who is moving out to the transition., its seems like Juan has a easy way to convince us to do all things.

**Comment by Dean13b** [comment]
Awaiting FS approval

**Comment by koushik-ac** [comment]
FS and TS were resent to Rafa and Thais yesterday. We have answered their additional questions. Awaiting their response now.

**Comment by koushik-ac** [comment]
Rafa has come back with additional comments. We have to reply to him today with necessary inputs.

**Comment by mbharathm** [comment]
Pending alignment with Rafa. Juan is assigned to take up the discussion with Rafa along with BES team.
The use case will be on hold until approval from Rafa. Will be followed up with Juan during the week of Oct 3-6.
Cc: @koushik-ac

**Comment by koushik-ac** [comment]
Aligned with Juan on Call. We need to make some changes to the PDD and get approval from Rafa and team and then start the development

**Comment by koushik-ac** [comment]
The automation is LIVE in Production and will be under Hyper Care for the next two weeks.

**Comment by Dean13b** [comment]
Queries raised due to data change in accounting document. To be resolved by running the bot with expected values.

**Comment by koushik-ac** [comment]
The production run has been paused and also the workflow has been deactivated. Once we receive PDD sign-off from the team for the new changes, we will begin the development and testing activities and then move them to Production. Only after that the workflow will be re-activated and schedule will be resumed.

**Comment by priyankag06** [comment]
@koushik-ac , could you please update the current status?

**Comment by koushik-ac** [comment]
The users have not yet replied to the queries or provided the approval to the changes as part of PDD V1.1. We have setup a Follow-up call with them for today(10/11/2023).

**Comment by koushik-ac** [comment]
End users are not available to provide the sign-off. We are following up with the backup user to receive the approval and start testing.

**Comment by koushik-ac** [comment]
We have received PDD sign-off for the new changes. Layout confirmation is still pending. We have started testing the changes in Dev system. Expected UAT for these changes next week.

**Comment by koushik-ac** [comment]
All the changes have been completed. SIT is successful and we have completed the UAT Demo as well. We are awaiting the UAT Approval to move this solution to Production.

**Comment by mbharathm** [comment]
@koushik-ac , I see a response from Christopher on performing a new execution. I will ask him to trigger automation. What I wanted to check is the usmas304 dependency as this might block some time on the other executions. Let me know your thoughts.

**Comment by koushik-ac** [comment]
Yes @mbharathm - I see one execution triggered yesterday and it has failed. Since I was testing A&P Woody and GRIR I did not re-trigger this failed execution. We can perform one end to end execution again today and share the results to Christopher.
